tests/transactions/btc.test.ts

```diff
@@ -0,0 +1,76 @@
+import { describe, expect, it } from 'vitest';
+
+import BigNumber from 'bignumber.js';
+import { selectUtxosForSend } from '../../transactions/btc';
```

Comment:
👏  for starting tests on this file.

I'd also consider unit testing each of the functions. smaller units for testing means other devs will debug faster

Reply:
I hate exporting functions for the sake of testing 😅 They'll also end up in the module export since we're doing export * from; though they are definitely complex enough to require their own tests 🤔

Reply:
Do you think it makes sense to pull these out into a file like brc20.utils.ts and then we can have tests for that file but not export the functions in this file? Then they won't get into the module export.

Reply:
ahh I see. yeah that makes sense. I see you did that already